beacon-chain/blockchain/service.go

```diff
@@ -471,7 +471,7 @@ func (c *ChainService) blockChildren(block *pb.BeaconBlock, state *pb.BeaconStat
 	if err != nil {
 		return nil, fmt.Errorf("could not tree hash incoming block: %v", err)
 	}
-	startSlot := block.Slot
+	startSlot := block.Slot + 1
```

Comment:
hmmm??

Reply:
sorry I missed this question! 

`+1` because current slot is included and we want the children of current slot. The will work without `+1` but it's less intuitive for people to read the code to figure out what's going on

ex: the following chain will show up as [a, c, b, d] and we only need to check if [c,b,d]'s parent root is equal to a's root
```
    /- c
a -----b 
   \-------d
```